README.md

```diff
@@ -36,8 +36,7 @@ There are two UNIX domain sockets used by the node-driver-registrar:
 * CSI driver socket:
   * Used by kubelet to interact with the CSI driver.
   * Created by the CSI driver.
-  * Exposed on a Kubernetes node via hostpath somewhere other than the Kubelet.
-    plugin registry (typically `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock`).
+  * Exposed on a Kubernetes node via hostpath somewhere other than the Kubelet plugin registry. (typically `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock` though a path outside of `/var/lib/kubelet/plugins/` will avoid to yield harmless error messages on the kubelet, for instance `/var/lib/kubelet/csi-plugins/<drivername.example.com>/csi.sock`).
```

Comment:
The "typically `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock`" is what we want to get rid of. I think it's better to not mention this at all or put it into a footnote ("For historic reasons, some CSI drivers use `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock`. This has the disadvantage that kubelet will attempt to connect to a plugin via this socket, which is not supported by a CSI driver and which therefore will lead to warning messages in the kubelet log.")

Reply:
Hmm from what I understand, kubelet is leaning towards not scanning subdirectories in the plugins dirs, which would then eliminate this problem. Apart from that, are there any other reasons for moving the socket out of the current dir? If so any official dir?

Reply:
> Hmm from what I understand, kubelet is leaning towards not scanning subdirectories in the plugins dirs, which would then eliminate this problem.

If that's the case, then why did @Madhu-1 get the following error?
```
Oct 31 13:57:11 foobar-csi-test-minion-group-xxx kubelet[1987]: E1031 13:57:11.226647    1987 plugin_watcher.go:115] error failed to get plugin info using RPC GetInfo at socket /var/lib/kubelet/plugins/com.google.csi.gcepd/csi.sock, err: rpc error: code = Unimplemented desc = unkno
wn service pluginregistration.Registration when handling create event: "/var/lib/kubelet/plugins/com.google.csi.gcepd/csi.sock": CREATE
```

> Apart from that, are there any other reasons for moving the socket out of the current dir? If so any official dir?

I'm not aware of any official recommendation. The closest to that is probably the documentation that you are currently changing...

Reply:
> If that's the case, then why did @Madhu-1 get the following error?
I guess because it's not implemented yet :sweat_smile: 

But yep, I'll change the wording